sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/ChangeFeedProcessorTest.java

```diff
@@ -424,6 +427,46 @@ public void readFeedDocumentsAfterSplit() {
         receivedDocuments.clear();
     }
 
+    @Test(groups = { "simple" }, timeOut = CHANGE_FEED_PROCESSOR_TIMEOUT)
+    public void testServiceItemLeaseSerialization() {
+        ZonedDateTime timeNow = ZonedDateTime.now();
+        String timeNowValue = timeNow.toString();
+
+        Lease lease1 = new ServiceItemLease()
+            .withId("id1")
+            .withLeaseToken("1")
+            .withETag("etag1")
+            .withOwner("Owner1")
+            .withContinuationToken("12")
+            .withTimestamp(timeNow)
+            .withTs("122311231");
+
+        Lease lease2 = new ServiceItemLease()
+            .withId("id2")
+            .withLeaseToken("2")
+            .withETag("etag2")
+            .withContinuationToken("22")
+            .withTimestamp(timeNow)
+            .withTs("122311232");
+
+        ObjectMapper mapper = new ObjectMapper();
+
+        try {
+            assertThat(mapper.writeValueAsString(lease1)).isEqualTo(
+                String.format("%s%s%s",
+                    "{\"id\":\"id1\",\"_etag\":\"etag1\",\"LeaseToken\":\"1\",\"ContinuationToken\":\"12\",\"timestamp\":\"",
+                    timeNowValue,
+                    "\",\"Owner\":\"Owner1\"}"));
+            assertThat(mapper.writeValueAsString(lease2)).isEqualTo(
+                String.format("%s%s%s",
+                    "{\"id\":\"id2\",\"_etag\":\"etag2\",\"LeaseToken\":\"2\",\"ContinuationToken\":\"22\",\"timestamp\":\"",
+                    timeNowValue,
+                    "\",\"Owner\":null}"));
+        } catch(Exception ex) {
```

Comment:
please drop the try/catch block here. if there is a failure the failure should get thrown and the exception needs to be added to the test method signature.

Reply:
done